src/goto-instrument/cover_instrument.h

```diff
@@ -16,6 +16,7 @@ Author: Peter Schrammel
 
 #include <goto-programs/goto_model.h>
 #include <util/message.h>
+#include "cover_basic_blocks.h"
```

Comment:
Changing the forward declaration below to cover_blocks_baset should be sufficient.